src/libraries/System.Text.Json/src/System/Text/Json/Serialization/JsonSerializer.Read.String.cs

```diff
@@ -48,7 +47,7 @@ public static partial class JsonSerializer
                 throw new ArgumentNullException(nameof(json));
             }
 
-            return Deserialize<TValue>(json.AsSpan(), typeof(TValue), options);
+            return ReadUsingOptions<TValue>(json.AsSpan(), typeof(TValue), options);
```

Comment:
Nit: perhaps pick a method name that better emphasizes the fact that this is deserialization that doesn't use source generated contracts?

Reply:
Good idea. Will think of a better name & tack onto another PR to avoid resetting CI here.